docs/rhte-apac-ch0.adoc

```diff
@@ -0,0 +1,82 @@
+Pre Requisites that need to be done on the AWS account
+
+* Created rhte-tower user
+* Created tower group
+ - permissions include
+ ** AmazonEC2FullAccess
+ ** AmazonS3FullAccess
+ ** AmazonS3FullAccess
+* Create a key pair
+
+Login to Tower host
+
+Create AWS user credentials
+
+* Top right click the "Settings" gear icon.
+* Click the "Credentials" pane.
+* Click "+ADD".
+* Provide a name of "AWS Credential".
+* Choose a type of "Amazon Web Services".
+* Copy in the "ACCESS KEY", and the "SECRET KEY".
+* Click "Save".
+
+Create AWS machine credentials
+
+* Click "+ADD".
+* Provide a name of "AWS Machine".
+* Choose a type of "Machine".
+* Provide a "USERNAME" of "ec2-user".
+* Paste in the following private key.
+* Click "Save"
+
+Create Inventory
+
+* Click "INVENTORIES" on the top navigation pane.
+* Click "+ADD".
+* Provide a name of "OpenShift Inventory"
+* Click "ADD GROUP"
+* Provide a name of "AWS"
+* Choose a "SOURCE" of "Amazon EC2"
+* Search for the "CLOUD CREDENTIAL" and select "AWS Credential" and click "SELECT"
+* Provide a "REGIONS" of "US East Ohio"
+* Provide the following "INSTANCE FILTER" of "tag:lab_user=<YOUR NAME HERE>"
+* Select "Overwrite", and "Update on Launch"
+* Click "SAVE"
+
+TIME: 8:11  into video
+
+* Add the following variables to the "SOURCE VARIABLES" of the inventory script
+ - hostname_variable: tag_Name
+
+
+Create Project
+
+* Click "PROJECTS" in the top navigation pane.
+* Click "ADD".
+* Chooset "SCM TYPE" of "Git".
+* Provide "SCM URL" of "https://github.com/sabre1041/managing-ocp-install-beyond.git" with a "SCM BRANCH" of "master".
+* Select "Clean" in the "SCM UPDATE OPTIONS",
+* Click "SAVE"
+* On the bottom navigation pane, click the cloud icon which triggers an SCM update.
+
+
+Create Template
+
+* Click "TEMPLATES" on the top navigation pane.
+* Click "+ADD", select "Job Template"
+* Provide a name of "Provision OCP Nodes"
+* Click the "SEARCH" icon for the "INVENTORY" input box and select "OpenShift Inventory" and then click "SELECT".
+* Click the "SEARCH" icon for the "PROJECT" input box and select "RHTE" and then click "SELECT".
+* Click the "Choose a playbook" in the "PLAYBOOK" input box and select the "openshift-infra/create_hosts.yml" playbook.
+* Click the "SEARCH" icon for the "MACHINE CREDENTIAL" input box and select "AWS Machine" and then click "SELECT".
+* Click the "SEARCH" icon for the "SELECT CLOUD CREDENTIAL" input box and select "AWS Credential" and then click "SELECT".
+* Click the "ADD SURVEY" button towards the top of the screen.
+ - Provide a "PROMPT" of "Student ID".
+ - Provide an "ANSWER VARIABLE NAME" of "student_id".
+ - Choose an "ANSWER TYPE" of "Text".
+ - Click "ADD".
+ - Click Save.
+* Add the following to the "EXTRA VARIABLES" pane.
+ - ec2_access_key: "{{ lookup('env', 'AWS_ACCESS_KEY') }}"
+ - ec2_secret_key: "{{ lookup('env', 'AWS_SECRET_KEY') }}"
```

Comment:
To be clear, these two variables can be collected via a survey (instead of putting them here in extra_variables). This only applies to student_id and lab_user.